kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1295,6 +1295,14 @@ object KyuubiConf {
       .timeConf
       .createWithDefault(Duration.ofSeconds(120).toMillis)
 
+  val ENGINE_ALIVE_CLOSE_CONNETION: ConfigEntry[Boolean] =
+    buildConf("kyuubi.session.engine.alive.closeConnection")
```

Comment:
kyuubi.session.engine.alive. closeConnectionOnTimeout?

Reply:
Sure, let me fix it.